lib/iris/tests/unit/concatenate/test_concatenate.py

```diff
@@ -242,6 +242,14 @@ def test_datatype_difference_message(self):
         with self.assertRaisesRegex(ConcatenateError, exc_regexp):
             _ = concatenate([cube_1, cube_2], True)
 
+    def test_dim_coords_overlap_message(self):
+        cube_1 = self.cube
+        cube_2 = cube_1.copy()
+        cube_2.coord("time").points = np.arange(1, 3, dtype=np.float32)
+        exc_regexp = "Dimensional coordinate extents differ"
```

Comment:
This will have to change to match the new error message.

Reply:
Done (the message must be called as a list to appear, otherwise the test doesn't work)